File: YuanChengXu/1016素数和/素数和.py
def is_prime(i):
    for j in range(2, i):
            if i % j == 0:
                return False
    return True
sum = 0
n = int(input("Enter a number between 100 and 10000:"))
if 100 < n < 10000:
    for i in range(101, n + 1):
        if is_prime(i):
            sum += i
    print(sum)

File: YuanChengXu/1016素数和/素数和.cpp
#include <iostream>

using namespace std;

int is_prime(int i);

int main()
{
	int sum = 0;
	int n, i;
	cin >> n;
	if(n > 100 && n < 10000)
	{
		for(i = 100; i < n + 1; i++)
		{
			if(is_prime(i))
			{
				sum += i;
			}
		}
		cout << sum << endl;
	}
}

int is_prime(int i)
{
	for(int j = 2; j < i; j++)
	{
		if(i % j == 0)
			return 0;
	}
	return 1;
}